archunit-junit/junit5/engine/build.gradle

```diff
@@ -2,6 +2,12 @@ plugins {
     id 'archunit.java-release-conventions'
 }
 
+repositories {
+    flatDir {
+        dirs "${rootDir}/inlineDeps/"
```

Comment:
Needed for the custom version of Surefire as well as one of the implementation Jars from Gradle. 

Once the relevant PRs are resolved, this flatDir repo will no longer be needed